docs/team/raynertjx.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Rayner Toh Jing Xiang's Project Portfolio Page
+---
+
+### Project: TAManager
+
+TAManager is a desktop address book application made for professors to manage their teaching assistants (TA) under their charge. It provides fast access to TAs' contact details and their availabilities for tutorials/labs. You can track teaching and claimable hours among your TAs and find relief TAs for impromptu events or unforeseen circumstances.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Modified the FindCommand that allows you to search for TAs.
+    * What it does: Allows the user to search for TA by name, course or free time.
+    * Justification: A professor may want to quickly retrieve a TA of a specific name, fetch a list of TAs teaching a certain course or find who's available during a certain time period.
+    * Highlights: This required me to tweak the parsing of the FindCommand to allow for the TAs to be searched by name, course or free time.
+
+* **New Feature**: Added the courses.json file to hold lesson data for the respective courses.
+    * What it does: Stores the lesson data for the respective courses in JSON format.
+    * Justification: This ensures that the lesson data for courses is stored in a human editable file format.
+    * Highlights: Professors can edit the courses.json file directly to store more lesson data for courses they're teaching.
```

Comment:
I think for this highlight it would be good to mention the challenges or the process in making this new feature.

Reply:
Good suggestion, I've made some changes